ajax/libs/across-tabs/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "across-tabs",
+  "description": "Easy communication between cross-origin browser tabs",
+  "filename": "across-tabs.min.js",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/wingify/across-tabs.git"
+  },
+  "keywords": [
+    "across-tabs",
+    "browser tabs",
+    "tabs communication",
+    "tabs postmessage",
+    "postmessage",
+    "webpack",
+    "es6",
+    "umd"
```

Comment:
No specific reason, I was planning to remove it from the original repo too. But for now, I have added it again.